extensions/amp-story/1.0/amp-story-bookend.css

```diff
@@ -202,6 +202,44 @@
   margin-right: 32px !important;
 }
 
+.i-amphtml-story-bookend-cta-link-wrapper {
+  display: flex !important;
+  flex-wrap: wrap !important;
+  box-sizing: border-box !important;
+  color: inherit !important;
+  overflow: hidden !important;
+  flex: 1 0 320px !important;
+  justify-content: space-between !important;
+  margin: 24px 0 !important;
+  padding: 0 32px !important;
+}
+
+.i-amphtml-story-bookend-cta-link {
+  background-color: hsla(0,0%,100%,0.15) !important;
+  border-radius: 100px !important;
+  overflow: hidden !important;
+  flex-grow: 0.4 !important;
+  text-decoration: none !important;
+  width: 45% !important;
```

Comment:
Where does `45%` come from?

Reply:
It is what most closely resembles the mocks and stays responsive, but we can change it to a px measurement.